### DISCUSSION TITLE
Moving searchBar position in Navbar

### DISCUSSION THREAD

**Discussion by Karan-S-dev** [facebook/docusaurus#7971] [Q&A] [answered]
Currently my searchbar is located on the right end of the navbar. 

But I want my login button to be present at the right end of my navbar and search bar to be present at the left side of login button. 

Is there a way to move the position of searchbar in navbar?

**Comment by slorber** [comment, marked as answer]
There's a search navbar item type that you can decide put in a different side (left/right) and in a different position than the default one
